packages/neon/neon/lib/src/widgets/error.dart

```diff
@@ -46,6 +58,11 @@ class NeonError extends StatelessWidget {
   /// Defaults to the nearest [IconTheme]'s [ColorScheme.error].
   final Color? color;
 
+  /// The display mode of this widget.
+  ///
+  /// Defaults to `NeonErrorType.column`.
```

Comment:
Shouldn't this be in the constructor because the default value is set in there? Then you can also see this information when looking at the widget in your code directly